### ISSUE TITLE
Remember-me does not work anymore since we migrated to Spring Security 4

### ISSUE THREAD

**Issue by jdubois** [jhipster/generator-jhipster#1366]
Only for stateful authentication, of course.

**Comment by jdubois** [comment]
This also makes the "session" screen fails, and is annoying when doing tests!!

**Comment by drissamri** [comment]
I'm not so familiar with this part of Spring Security, but with some debugging I found out that the remember-me form request parameter changed in Spring Security 4 from **_spring_security_remember_me** to **remember-me** 

I've changed the form parameter and explicitly set the parameter name in the SecurityConfiguration so if this changes again in the future it won't have impact anymore. Now the session screen seems to work again. 

Let me know if there are any other issues with this.

More information can be found here:
Docs: http://docs.spring.io/spring-security/site/migrate/current/3-to-4/html5/migrate-3-to-4-xml.html#m3to4-xmlnamespace-remember-me
GitHub: https://github.com/spring-projects/spring-security/commit/5f57e5b0c3726466db4f5d0521ac26423f0d9cd4)
